jerry-debugger/jerry_client.py

```diff
@@ -279,7 +279,6 @@ def main():
 
         protocol = Socket(address)
     elif args.protocol == "serial":
-        from jerry_client_serial import Serial
```

Comment:
Do we really need to move this out of here? If so, we always expect that a user will install the serial package even if they will only use the socket based communication.

Reply:
Moved import to top level because of pylint.

************* Module jerry_client
jerry-debugger/jerry_client.py:281:8: C0415: Import outside toplevel (jerry_client_serial) (import-outside-toplevel)

Reply:
I can move back the import there and disable pylint on that row.

Reply:
This was only error locally so I reverted this.